e2e/nx-init/src/nx-project-graph.test.ts

```diff
@@ -12,7 +12,7 @@ describe('project graph creation', () => {
 
   afterEach(() => cleanupProject());
 
-  it('should correctly build the nxdeps.json containing files for the project', () => {
+  it('should correctly build graph.json containing files for the project', () => {
```

Comment:
I know its an implementation detail, but do we write the original file still? I don't think its terribly uncommon for people to parse this file during CI, even if its not something we would recommend

Reply:
We don't write it anymore